evaluation.py

```diff
@@ -158,29 +158,38 @@ def _non_max_supression(self, plain, threshold, windowSize=3):
 
 x,y,w,h : {int or float} optional bounding box info, anchored at top left of image
 """
-def load_and_preprocess_img(img_path, num_hg_blocks, x=None, y=None, w=None, h=None):
+def load_and_preprocess_img(img_path, num_hg_blocks, bbox=None):
     img = Image.open(img_path).convert('RGB')
 
-    if x is None or y is None or w is None or h is None:
-        # If any of the parameters are missing, crop a square area from top left of image
-        smaller_dim = img.size[0] if img.size[0] <= img.size[1] else img.size[1]
-        bbox = [0, 0, smaller_dim, smaller_dim]
-    else:
+    # Required because PIL will read EXIF tags about rotation by default. We want to
+    # preserve the input image rotation so we manually apply the rotation if required.
+    # See https://stackoverflow.com/questions/4228530/pil-thumbnail-is-rotating-my-image/
+    # and the answer I used: https://stackoverflow.com/a/63798032
+    img = ImageOps.exif_transpose(img)
```

Comment:
This sounds good, have you tested to see if it changes the bbox in anyway? (Just thinking about the need to reverse)

Reply:
I haven't but I think we're good here because:
1. This is put in to ensure that the image we feed to the model is the right orientation (the one we see if we open it in a file viewer). I'd say it's safe to assume that the bounding box is created respective to the right orientation. 
2. We might not need this change in data generator which is what is used to train the model.